runtime/flang/CMakeLists.txt

```diff
@@ -8,7 +8,11 @@ enable_language(C ASM Fortran) # Enable assembly and Fortran
 
 SET(ASM_OPTIONS "-DLINUX_ELF")
 SET(CMAKE_ASM_FLAGS "${CFLAGS} ${ASM_OPTIONS}" )
-SET(CMAKE_SHARED_LINKER_FLAGS "-no-flang-libs")
+if (NOT MSVC)
```

Comment:
Maybe it's better to check WIN32 instead of MSVC. We could use clang with regular front end on Windows which detected as GNU compiler.

Reply:
...but, is the linker issue windows specific or rather MSVC specific?

Reply:
All of these are MSVC specific and doesn't affect MinGW.
CMake will identify clang with target triple `x86_64-windows-msvc` as MSVC and clang with target triple `x86_64-w64-mingw32` as MinGW.